database/transaction_test.go

```diff
@@ -194,7 +194,7 @@ func TestTxTable(t *testing.T) {
 
 		// Renaming a non existing table should return an error
 		err = tx.AddField("bar", fieldToAdd)
-		require.EqualError(t, database.ErrTableNotFound, err.Error())
+		require.True(t, errors.Is(err, database.ErrTableNotFound))
```

Comment:
For consistency with other tests that expect `database.ErrTableNotFound`. This also gives us a better error messages since it prints actual error instead of something like “expected true got false”.

```suggestion
		if !errors.Is(err, database.ErrTableNotFound) {
			require.Equal(t, err, database.ErrTableNotFound)
		}
```